src/Compatibility/GenAPI/Microsoft.DotNet.GenAPI.Task/GenAPITask.cs

```diff
@@ -62,17 +64,21 @@ public class GenAPITask : TaskBase
         /// <inheritdoc />
         protected override void ExecuteCore()
         {
+            (IAssemblySymbolLoader loader, Dictionary<string, IAssemblySymbol> assemblySymbols) = AssemblyLoaderFactory.CreateFromFiles(
+                assembliesPaths: Assemblies ?? throw new NullReferenceException("Assemblies cannot be null."),
+                assemblyReferencesPaths: AssemblyReferences,
+                RespectInternals);
```

Comment:
One of the main design principles in the Compatibility tools is to keep the UI layer as thin as possible and let the business logic layer handle with the start-up and shutdown. The existing code base does this well, it just calls into GenAPI.Run without calling into lower level types like inside `Microsoft.CodeAnalysis` or `ApiSymbolExtensions`.